src/EnergyPlus/ThermalComfort.cc

```diff
@@ -2957,7 +2957,7 @@ namespace ThermalComfort {
 
         // SUBROUTINE PARAMETER DEFINITIONS:
         static Real64 constexpr alpha(0.8);
-        static Array1D<Real64> const alpha_pow({pow_6(alpha), pow_5(alpha), pow_4(alpha), pow_3(alpha), pow_2(alpha), alpha, 1.0}); // alpha^(7-i)
+        static constexpr std::array<Real64, 7> alpha_pow = {0.262144, 0.32768, 0.4096, 0.512, 0.64, 0.8, 1.0}; // alpha^(6-0)
```

Comment:
I just hardcoded these values here. If `alpha` ever changes, this will also have to change. There are a few places like that throughout the code. The array history stuff at the top of HVACManager comes to mind, but there may be others.